warehouse/models/mart/gtfs/fct_vehicle_positions_messages.sql

```diff
@@ -29,7 +30,7 @@ fct_vehicle_positions_messages AS (
         base64_url,
         _extract_ts,
         _config_extract_ts,
-        _name,
+        _gtfs_dataset_name,
```

Comment:
does this need to be changed in the YAML too?

Reply:
yep ty